purchases/src/main/kotlin/com/revenuecat/purchases/SubscriptionInfo.kt

```diff
@@ -73,6 +75,22 @@ class SubscriptionInfo(
      * The transaction id in the store of the subscription.
      */
     val storeTransactionId: String?,
+    /**
+     * Date when the subscription will auto-resume.
+     */
+    val autoResumeDate: Date?,
+    /**
+     * The display name of the subscription.
+     */
+    val displayName: String?,
```

Comment:
This is the one set in the dashboard right?

Reply:
Right, it would be great to dissambiguate this somehow... Maybe could be `rcDisplayName`? Not sure TBH 😅

Reply:
Maybe we can just make it more clear in the docs?

Reply:
Yeah, rethinking about this, I think it's fine 👍